### PR TITLE
Update sdk/lib landing pages, add call to action button

### PR DESCRIPTION
**Pull request by nikhil-dabhade**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/fragments/lib/js.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+**Amplify JavaScript**  
+The Amplify JavaScript libraries are supported for different web and mobile frameworks including React, React Native, Angular, Ionic, and Vue. It is recommended you first complete the [Getting Started](../../start/start.md) guide for Amplify JavaScript.
+
+What next ? Here are some things to add to your app:
```
Comment (brene) [inline docs/fragments/lib/js.md]: Typo

File: docs/lib/lib.md [hunk 2]
```diff
@@ -5,9 +5,13 @@ disableTOC: true
...
 
-The libraries can be used independent of whether the backend created was created using the Amplify CLI. 
+**How are these different from the Mobile SDKs?**  
+The Amplify client libraries are use-case centric as compared to the Mobile SDKs which are service-centric. This enables you to focus on your use case more rather than figuring out the AWS service nuances. In most cases, the Mobile SDKs are low level auto-generated while the Amplify libraries are hand authored providing a highly abstracted category based programming model. You can also use the Mobile SDKs with the Amplify libraries using escape hatches if the use case you are trying to build is not available in Amplify libraries.
```
Comment (brene) [inline docs/lib/lib.md]: inconsistent writing of "use-case" sometimes with - sometimes without.

File: docs/fragments/lib/android-sdk.md [hunk 3]
```diff
@@ -1,4 +1,13 @@
-## AWS Mobile SDK for Android
-
+**AWS Mobile SDK for Android**  
 This guide shows how to build an app using AWS Mobile SDK for Android and the Amplify CLI toolchain.
-To use our new, preview developer experience with new Amplify Libraries for iOS, [click here](/lib).
+To use our new, preview developer experience with new Amplify Libraries for Android, [click here](~/lib/lib.md).
```
Comment (brene) [inline docs/fragments/lib/android-sdk.md]: Change the language to: "Click here to use our new preview developer experience..."

File: docs/fragments/lib/android.md [hunk 4]
```diff
@@ -1,4 +1,6 @@
-## Amplify Android (Preview)
+**Amplify Android (preview)**  
+The Amplify Android libraries are available in Java programming language. This guide shows how to build an app using our Amplify Libraries for Android (Preview) and the Amplify CLI toolchain. To use the existing AWS Mobile SDK for iOS instead, [click here](../sdk/sdk.md).
```
Comment (brene) [inline docs/fragments/lib/android.md]: in Java.

Comment (brene) [inline docs/fragments/lib/android.md]: Same here. The "Click here" sentence seems very gnarly. How about start the sentence with "Click here to..."

File: docs/fragments/lib/ios-sdk.md [hunk 5]
```diff
@@ -1,4 +1,13 @@
-## AWS Mobile SDK for iOS
-
+**AWS Mobile SDK for iOS**  
 This guide shows how to build an app using AWS Mobile SDK for iOS and the Amplify CLI toolchain.
-To use our new, preview developer experience with new Amplify Libraries for iOS, [click here](/lib).
+To use our new, preview developer experience with new Amplify Libraries for iOS, [click here](~/lib/lib.md).
```
Comment (brene) [inline docs/fragments/lib/ios-sdk.md]: Start the sentence with "click here"?

File: docs/fragments/lib/ios.md [hunk 6]
```diff
@@ -1,4 +1,6 @@
-## Amplify iOS (Preview)
+**Amplify iOS (preview)**  
+The Amplify iOS libraries are written in Swift programming language. This guide shows how to build an app using our Amplify Libraries for iOS (Preview) and the Amplify CLI toolchain. To use the existing AWS Mobile SDK for iOS instead, [click here](../sdk/sdk.md).
```
Comment (brene) [inline docs/fragments/lib/ios.md]: Start the sentence with "Click here"

File: docs/fragments/lib/android.md (hunk 4 above)
Comment (nikhil-dabhade) [reply docs/fragments/lib/android.md]: I have kept it consistent with the message which we have on our live site today: https://aws-amplify.github.io/docs/sdk/ios/start

File: docs/fragments/lib/ios-sdk.md (hunk 5 above)
Comment (nikhil-dabhade) [reply docs/fragments/lib/ios-sdk.md]: Same as previous comment

File: docs/fragments/lib/ios.md (hunk 6 above)
Comment (nikhil-dabhade) [reply docs/fragments/lib/ios.md]: Same as previous comment

**Pull request merged**